main.go

```diff
@@ -37,6 +38,7 @@ var (
 	useJournal       = kingpin.Flag("exim.use-journal", "Use the journal instead of log file tailing").Envar("EXIM_USE_JOURNAL").Bool()
 	syslogIdentifier = kingpin.Flag("exim.syslog-identifier", "Syslog identifier used by Exim").Default("exim").Envar("EXIM_SYSLOG_IDENTIFIER").String()
 	tailPoll         = kingpin.Flag("tail.poll", "Poll logs for changes instead of using inotify.").Envar("TAIL_POLL").Bool()
+	configFile       = kingpin.Flag("web.config", "Path to config yaml file that can enable TLS or authentication.").Default("").String()
```

Comment:
I haven't used the exporter-toolkit before. Should this be `web.config.file` per [flag.go](https://github.com/prometheus/exporter-toolkit/blob/master/web/kingpinflag/flag.go#L23)?

A number of users run the exporter under Docker or Kubernetes and can't control the command line arguments directly. We'll need to add `.Envar("WEB_CONFIG_FILE")` to support them.

Reply:
Ah, I was using [node_exporter](https://github.com/prometheus/node_exporter/blob/master/node_exporter.go#L165) as an example but `web.config.file` works. I remember seeing both flags around in other exporters maybe the name was updated at some point. 🤷

I've updated the flag name and added the Envar, tested it locally and both seem to work :)